tardis/model/tests/test_csvy_model.py

```diff
@@ -42,16 +42,48 @@ def test_compare_models(filename):
                 config_model_val = config_model_val.value
                 csvy_model_val = csvy_model_val.value
             npt.assert_array_almost_equal(csvy_model_val, config_model_val)
+    pd.testing.assert_frame_equal(csvy_model.abundance,config_model.abundance,check_less_precise = True)
```

Comment:
This are the lines that check that the models' abundances are equal. There was an issue were this dataframes were not compared and both implementations had different outputs.
This test both values and shapes of dataframes.